src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -625,6 +625,13 @@ public virtual IEnumerable<CallStackFrame> GetCallStack()
         public virtual Breakpoint GetBreakpoint(int id) =>
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-windows`
TestAppDomainProcessExitEvenHandlerNotLeaking
```powershell
Assert.DoesNotContain() Failure
Found:    (filter expression)
In value: Delegate[] [EventHandler { Method = Void DisposeOnShutdown(System.Object, System.EventArgs), Target = null }, EventHandler { Method = Void CurrentDomain_ProcessExit(System.Object, System.EventArgs), Target = null }]
   at PSTests.Sequential.RunspaceTests.TestAppDomainProcessExitEvenHandlerNotLeaking() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 127
```